moto/apigateway/responses.py

```diff
@@ -303,6 +303,54 @@ def restapis_authorizers(self, request, full_url, headers):
 
         return 200, {}, json.dumps(authorizer_response)
 
+    def request_validators(self, request, full_url, headers):
+        self.setup_class(request, full_url, headers)
+        url_path_parts = self.path.split("/")
+        restapi_id = url_path_parts[2]
+
+        try:
+            restApi = self.backend.get_rest_api(restapi_id)
+            if self.method == "GET":
+                validators = restApi.get_request_validators()
+                res = json.dumps(
+                    {"items": [validator.to_dict() for validator in validators]}
+                )
+                return 200, {}, res
+            if self.method == "POST":
+                name = self._get_param("name")
+                validateRequestBody = self._get_bool_param("validateRequestBody")
+                validateRequestParameters = self._get_bool_param(
+                    "validateRequestParameters"
+                )
+                validator = restApi.create_request_validator(
+                    name=name,
+                    validateRequestBody=validateRequestBody,
+                    validateRequestParameters=validateRequestParameters,
+                )
+                return 200, {}, json.dumps(validator)
+        except BadRequestException as e:
+            return self.error("BadRequestException", e.message)
+        except NotFoundException as e:
+            return self.error("NotFoundException", e.message)
```

Comment:
This should have a (stand alone) test case as well.
Negative tests like this should follow the following format:
https://github.com/spulec/moto/blob/master/tests/test_apigateway/test_apigateway.py#L78

Reply:
I think I just added this looking at other code, let me remove it for the time being.

Reply:
No, you were right adding it, actually, it should be there :) The `self.error()` method parses our custom exceptions into a JSON format that is understood by all AWS SDK's (Python/Java/.NET/etc)